src/Features/Core/Portable/ConflictMarkerResolution/AbstractConflictMarkerCodeFixProvider.cs

```diff
@@ -45,19 +44,15 @@ protected AbstractResolveConflictMarkerCodeFixProvider(string diagnosticId)
 
             while (true)
             {
-                token = token.GetNextToken(includeZeroWidth: true);
```

Comment:
moved to bottom of loop.  the <<<<<  might be immediately followed by ======, so we can't just immediately jump to the next token.